lib/peep.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :body, :time, :username
+
+  def initialize(id, body, time, username)
+    @id = id
+    @body = body
+    @time = time
+    @username = username
+  end
+
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
Peep class is nice but this big `chonk` of code appears twice, maybe it could be moved to a helper method?